openmetadata-ui/src/main/resources/ui/src/components/Ingestion/Ingestion.component.tsx

```diff
@@ -276,6 +276,30 @@ const Ingestion: React.FC<IngestionProps> = ({
     );
   };
 
+  const getAddIngestionName = (type: PipelineType): string => {
+    let name;
+    switch (type) {
+      case PipelineType.ElasticSearchReindex:
+        name = `${t('label.add')} ${startCase(type)}`;
```

Comment:
We shouldn't use `startCase(type)`
If we choose different lang it will still show English or you can say default, Use localisation here too